From 6c2c185e5270484fffa1f895c7c4420c6a15ff5d Mon Sep 17 00:00:00 2001
From: Anisetti Avinash Krishna <anisetti.avinash.krishna@intel.com>
Date: Mon, 20 Apr 2026 09:18:24 +0000
Subject: [PATCH] Enable staticlocation mixin

Add staticlocation: true to caas mixins configuration to enable the
static
location HAL feature for Celadon caas builds.

Tracked-On: NIACP3-1182
Signed-off-by: Anisetti Avinash Krishna <anisetti.avinash.krishna@intel.com>
---
 caas/mixins.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 73b5784c..2794be96 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -10,6 +10,7 @@ product.mk: device.mk
 kernel: gmin64(useprebuilt=false,src_path=kernel/lts2023-chromium, loglevel=7, interactive_governor=false, relative_sleepstates=false, modules_in_bootimg=false, external_modules=,debug_modules=, use_bcmdhd=false, use_iwlwifi=false, extmod_platform=bxt, iwl_defconfig=, cfg_path=config-lts/lts2023-chromium, more_modules=true,lts2023_chromium_src_path=kernel/lts2023-chromium, lts2023_chromium_cfg_path=config-lts/lts2023-chromium, linux_intel_lts2023_src_path=kernel/linux-intel-lts2023, linux_intel_lts2023_cfg_path=config-lts/linux-intel-lts2023,lts2024_android_src_path=kernel/lts2024-android,lts2024_android_cfg_path=config-lts/lts2024-android,linux_intel_lts2024_src_path=kernel/linux-intel-lts2024, linux_intel_lts2024_cfg_path=config-lts/linux-intel-lts2024)
 disk-bus: auto
 boot-arch: project-celadon(uefi_arch=x86_64,fastboot=efi,ignore_rsci=true,disable_watchdog=true,watchdog_parameters=10 30,verity_warning=false,txe_bind_root_of_trust=false,bootloader_block_size=4096,verity_mode=false,disk_encryption=false,file_encryption=true,metadata_encryption=true,fsverity=true,target=caas,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true,usb_storage=true,live_boot=true,userdata_checkpoint=true,data_use_f2fs=true,trusty=true)
+staticlocation: true
 sepolicy: enforcing
 bluetooth: btusb(ivi=false)
 audio: project-celadon
-- 
2.34.1

